pyqtgraph/parametertree/Parameter.py

```diff
@@ -221,6 +227,7 @@ def __init__(self, **opts):
         if 'default' not in self.opts:
             self.opts['default'] = None
             self.setDefault(self.opts['value'])
+        self._modifiedSinceReset = not self.valueIsDefault()
```

Comment:
I don't think this is right -- if the initial value happens to be the same as the default, then it would end up pinned. Seems like it should be explicitly reset to default for this to happen?